testing-modules/junit-5/src/test/java/com/baeldung/execution/time/SampleExecutionTimeUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.execution.time;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class SampleExecutionTimeUnitTest {
+
+    @Test
+    public void someUnitTest() {
+
+        assertTrue(this::doSomething);
```

Comment:
This seems needlessly complex. You're using a boolean supplier to `assertTrue`. Why not just call `doSomething()` directly?